static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -42,9 +43,26 @@ function ReplayTimeline({}: Props) {
   const chapterFrames = replay.getChapterFrames();
   const networkFrames = replay.getNetworkFrames();
 
+  // timeline size is 300% larger
```

Comment:
Larger than it originally was. This will be altered when implementing zoom, but I was thinking of making the scrolling timeline default to 3x zoomed in?